erp/mapper/vaccination.py

```diff
@@ -231,6 +267,7 @@ def _retrieve_commune_ext(self):
             )
 
         if not commune_ext:
+            print(self.erp.code_insee, self.erp.code_postal)
```

Comment:
Do we want to keep this print statement?

Reply:
I had a bug on communes, which i fact was due to some INSEE codes we don't have in DB (some exotic ones). Forgot to remove this.